glib/src/collections/list.rs

```diff
@@ -486,6 +486,25 @@ impl<T: TransparentPtrType> std::iter::IntoIterator for List<T> {
     }
 }
 
+impl<T: TransparentPtrType> std::iter::Extend<T> for List<T> {
```

Comment:
Could you add the same for `SList`? Basically same implementation

Reply:
Done